scripts/rageshake.go

```diff
@@ -0,0 +1,108 @@
+// Run a web server capable of dumping bug reports sent by Riot.
+// Requires Go 1.5+
+// Usage:   go run rageshake.go PORT
+// Example: go run rageshake.go 8080
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+var maxPayloadSize = 1024 * 1024 * 55 // 55 MB
+
+type LogEntry struct {
+	ID    string `json:"id"`
+	Lines string `json:"lines"`
+}
+
+type Payload struct {
+	Text      string     `json:"text"`
+	Version   string     `json:"version"`
+	UserAgent string     `json:"user_agent"`
+	Logs      []LogEntry `json:"logs"`
+}
+
+func respond(code int, w http.ResponseWriter) {
+	w.WriteHeader(code)
+	w.Write([]byte("{}"))
+}
+
+func gzipAndSave(data []byte, filepath string) error {
+	if _, err := os.Stat(filepath); err == nil {
```

Comment:
should we be specifying a directory as an explicit option rather than just using the cwd?